### PR TITLE
Updated ReadMe

### PR DESCRIPTION
**Pull request by jtatia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
+   3. [Documentation](#documentation)
+
+## Build
+### Prerequisites
+
+#### Java
+
+The module is a Java application which is why you need to install a Java JDK.
+
+If you want to build the master branch you will need Java 7.
```
Comment (wluyima) [inline README.md]: Are you sure you need java 7? Or did you just paste and commit? Please read through the entire file to make sure everything applies

Comment (jtatia) [reply README.md]: I didn't use anything that requires Java 8

Comment (wluyima) [reply README.md]: I'm not saying require java 8, the project where you copied this requires java 7, are you sure your module requires java 7, please read the file, cross check and make sure everything applies.

Comment (jtatia) [reply README.md]: Sorry, I'll probably have to test this properly.

Comment (wluyima) [reply README.md]: The question is what have you been using? And anyways you should require java 6

Comment (wluyima) [reply README.md]: Can you please address this?

**Pull request merged**